### PR TITLE
added announcement command, nodemon for testing

### PR DESCRIPTION
**Pull request by lsminter**
Added a command for the Staff to use to make announcements in the Announcement channel. I didn't make the command roll specific since I made the command channel-specific and only Staff can type in the Announcement channel.

The staff member types `!Announcement` and the eggo bot will dm you asking `What is your announcement?` You then reply to the bot in the dm channel with what you want to announce.

Screen Shot 2022-03-24 at 1 56 28 PM

The bot will delete the `!Announcement` command and then post your announcement while tagging `@everyone`. 

Screen Shot 2022-03-24 at 1 57 49 PM

I also added Nodemon to make testing much easier with the scripts for dev and starting the bot.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -57,6 +60,22 @@ client.on('message', (message) => {
...
+
+  if (message.content === '!Announcement' && message.channel.id === '953362393071775814'){
+    await message.author.send('What is your announcement?')
+
+    const filter = (m) => m.author.id === message.author.id
+    const userAnnouncement = await message.author.dmChannel.awaitMessages(filter, {max: 1})
+      .then((collected) => collected.first().content)
+    if (userAnnouncement === 'cancel'){
+      await message.author.send('Canceled the announcement.')
+      message.delete()
```
Comment (lsminter) [inline index.js:74]: If you decide that you want to cancel the announcement, just type `cancel` in the dm message and the bot will cancel it and let you know that the announcement was canceled.

File: index.js [hunk 2]
```diff
@@ -57,6 +60,22 @@ client.on('message', (message) => {
     message.reply(message.author.displayAvatarURL())
     ev.addField('avatar', true)
   }
+
+  if (message.content === '!Announcement' && message.channel.id === '953362393071775814'){
```
Comment (zacjones93) [inline index.js]: Might think about naming these id's, `const ANNOUNCEMENT_CHANNEL = 953362393071775814` (assuming it's the announcement channel) for clarity

**Review by zacjones93 — COMMENTED** [review]
Looks good, had one comment!

File: index.js [hunk 3]
```diff
@@ -63,7 +63,7 @@ client.on('message', async (message) => {
 
   const announcementChannel = '953362393071775814'
 
-  if (message.content.toLocaleLowerCase() === '!announcement' && message.channel.id === announcementChannel){
+  if (message.content.toLowerCase() === '!announcement' && message.channel.id === announcementChannel){
```
Comment (lsminter) [inline index.js:66]: Added in that announcement channel variable and had the command be lower case. Figured this way there wouldn't be any issue if the Staff member did `!Announcement` or `!announcement`. Just to help avoid any confusion.

**Review by zacjones93 — APPROVED** [review]

**Review by zacjones93 — APPROVED** [review]

**Pull request merged**